conan/tools/qbs/qbs.py

```diff
@@ -45,6 +63,12 @@ def _get_common_arguments(self):
         ]
 
     def resolve(self, parallel=True):
+        '''
+        Wraps the ``qbs revolve`` call.
+        If QbsDeps generator is used, this will also set the necessary properites of the Qbs
+        "conan" module provider automatically adding dependencies to the project.
+        :param parallel: Whether to use multi-threaded resolving. Defaults to ``True``.
+        '''
         args = self._get_common_arguments()
 
         if parallel:
```

Comment:
Sorry, I know it's unrelated but found it while reading the class - Why is the parallel option needed here? The build method always uses njobs from the user conf, but for that to happen here, you need to pass parallel=True first, instead of relying on the value of the conf. Do you have any insight into why this is like that? Thanks!

Reply:
Long story short - we obviously used parallel **build** since ages, however project resolving aka configuration was done in a single thread. We recently added a parallel resolving, but it is known that there is a threading issue somewhere that (rarely) causes a crash; so we offer an option to fallback to single-threaded resolving just in case.

Reply:
Thanks! Makes lots of sense to then limit it :)